prow/integ-suite-kind.sh

```diff
@@ -171,6 +171,16 @@ fi
 if [[ -z "${SKIP_BUILD:-}" ]]; then
   trace "setup kind registry" setup_kind_registry
   trace "build images" build_images "${PARAMS[*]}"
+  WASM_ATTRGEN_TAG="359dcd3a19f109c50e97517fe6b1e2676e870c4d"
```

Comment:
maybe we should copy this to some static tag so we don't need to keep it in sync with 
```
tests/integration/telemetry/api/customize_metrics_test.go
111-                    }
112-            })
113-}
114-
115-func setupWasmExtension(t framework.TestContext) {
116:    proxySHA := "359dcd3a19f109c50e97517fe6b1e2676e870c4d"
```

Reply:
Done.